packages/kbn-eslint-plugin-disable/src/rules/no_protected_eslint_disable.test.ts

```diff
@@ -244,6 +241,26 @@ for (const [name, tester] of [tsTester, babelTester]) {
             const a = 1;
           `,
         },
+        {
+          filename: 'foo.ts',
+          code: dedent`
+            /* eslint-disable no-var, @kbn/disable/no_protected_eslint_disable */
+            const a = 1;
+          `,
+          errors: [
+            {
+              line: 1,
+              messageId: PROTECTED_DISABLE_MSG_ID,
+              data: {
+                disabledRuleName: '@kbn/disable/no_protected_eslint_disable',
+              },
+            },
+          ],
+          output: dedent`
+            /* eslint-disable no-var */
+            const a = 1;
+          `,
+        },
```

Comment:
Sorry, mind adding a test where the first and middle rule are removed?

Reply:
Sure, added 👍